src/features/metadata/components/BuildDropdown.tsx

```diff
@@ -6,22 +6,25 @@ import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 import IconButton from "@mui/material/IconButton";
 import { useAppDispatch } from "../../../hooks";
 import { currentBuildIdChanged } from "..";
+import { Build } from "../../../common/models";
+import { buildDatetimeStatus } from "../../../utils/helpers/buildMapper";
 
 interface IBuildProps {
   /**
    * @param builds list of builds
    * @param currentBuildId id of the current build
-   * @param onChangeStatus update the build status
+   * @param selectedBuildId id of the build selected from the dropdown
    */
-  builds: {
-    id: number;
-    name: string;
```

Comment:
These mapped builds were kind of masking a dependency on `currentBuildId`, which is used in the dropdown options to label the environment's default, or current, build as "Active"